pkg/pquerier/collector.go

```diff
@@ -110,14 +121,14 @@ func rawCollector(ctx *selectQueryContext, res *qryResults) {
 
 func aggregateClientAggregates(ctx *selectQueryContext, res *qryResults) {
 	ctx.logger.Debug("using Client Aggregates Collector for metric %v", res.name)
-	it := newRawChunkIterator(res, nil)
+	it := newRawChunkIterator(res, ctx.logger.GetChild("rawChunkIterator"))
```

Comment:
This (`GetChild`) is generally done inside the constructor. See [nuclio](https://github.com/nuclio/nuclio/blob/f2ebdd95e95c4a1cb24fcb14e18ea88cd9093eba/pkg/processor/trigger/kinesis/trigger.go#L42) for an example.